app/code/Bulbulatory/Recommendations/Controller/Adminhtml/Table/Index.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace Bulbulatory\Recommendations\Controller\Adminhtml\Table;
+
+class Index extends \Magento\Backend\App\Action
+{
+    protected $resultPageFactory = false;
+
+    public function __construct(
+        \Magento\Backend\App\Action\Context $context,
+        \Magento\Framework\View\Result\PageFactory $resultPageFactory)
+    {
+        parent::__construct($context);
+        $this->resultPageFactory = $resultPageFactory;
+    }
+    public function execute()
+    {
+        $resultPage = $this->resultPageFactory->create();
+        //$resultPage->setActiveMenu('Bulbulatory_Recommendations::bulbulatory_recommendations_table');
+        $resultPage->getConfig()->getTitle()->prepend((__('Table')));

```

Comment:
`Table` title doesn't really say much about what table it is, it would be better to name it `Recommendations List` or `Recommendations Table` or something like that.

Reply:
@TM18 

> Table title doesn't really say much about what table it is, it would be better to name it Recommendations List or Recommendations Table or something like that.

Fixed! I called it "Customer Recommendations List"